runtime/build.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+use deno_core::JsRuntime;
+use deno_core::RuntimeOptions;
+use std::env;
+use std::path::Path;
+use std::path::PathBuf;
+
+fn create_snapshot(
+  mut js_runtime: JsRuntime,
+  snapshot_path: &Path,
+  files: Vec<PathBuf>,
+) {
+  deno_web::init(&mut js_runtime);
+  deno_fetch::init(&mut js_runtime);
+  deno_crypto::init(&mut js_runtime);
+  // TODO(nayeemrmn): https://github.com/rust-lang/cargo/issues/3946 to get the
+  // workspace root.
+  let display_root = Path::new(env!("CARGO_MANIFEST_DIR")).parent().unwrap();
+  for file in files {
+    println!("cargo:rerun-if-changed={}", file.display());
+    let display_path = file.strip_prefix(display_root).unwrap();
+    let display_path_str = display_path.display().to_string();
+    js_runtime
+      .execute(
+        &("deno:".to_string() + &display_path_str.replace('\\', "/")),
+        &std::fs::read_to_string(&file).unwrap(),
+      )
+      .unwrap();
+  }
```

Comment:
There is now a lot of duplicated logic between cli/build.rs and runtime/build.rs

Ideally these could be factored out into deno_core so they can be shared. I think it would be fine to postpone that work, but maybe add some TODOs.

Reply:
I'll add TODO